core/src/main/scala/scalapb/zio_grpc/CallContext.scala

```diff
@@ -0,0 +1,12 @@
+package scalapb.zio_grpc
+
+import io.grpc.MethodDescriptor
+import io.grpc.CallOptions
+
+import zio.UIO
+
+final case class CallContext(
```

Comment:
Can you rename the class and the file to `ClientCallContext`, just so it's obvious when seeing it for the first time (or after a long break) what part of the code it participates in.

Reply:
Done